ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +249,79 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    @staticmethod
+    def drop_unsupported_parameters(
```

Comment:
This should be a generic solution, for any condition.
Conditions may have one or more internal conditions.
These conditions are mapped to the cluster conditions.
Please use `python-benedict` to get the value from the cluster instance.

Reply:
Done